### ISSUE TITLE
Syntax for adding aliases

### ISSUE THREAD

**Issue by ajshea** [adigitalife/yourls-allow-aliases#6]
The Readme should be updated to show the correct syntax for adding aliases. The syntax is not mentioned at all, and aliased domains won't work with this plugin unless they are added in the config. 

`` define( 'YOURLS_SITE', 'http://yourdomain.com' , 'http://aliaseddomain.com' );

This would not work for me until I found the correct syntax in a comment at the old Google Code repository: 
https://code.google.com/archive/p/yourls-allow-aliases/issues/1

**Comment by mauamolat** [comment]
Does this mean that it let us use multiple domains for certain 1 instance of YOURLS?

**Comment by ajshea** [comment]
Yes, but the domains all must resolve/point to the same directory. That's why they are called "aliases". 

I use it so that YOURLS links work with www.domain.com and domain.com
